<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user_dialog</class>
 <widget class="QDialog" name="user_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1267</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tab_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1161</width>
     <height>651</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color : #181DA4 ;
border-radius : 20px ;
color : white ;
}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="Home">
    <attribute name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/new/prefix1/30.png</normaloff>:/new/prefix1/30.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Home</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>20</y>
       <width>531</width>
       <height>481</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>4.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>521</width>
       <height>291</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>14.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>931</width>
       <height>541</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>13.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_stat">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>260</y>
       <width>211</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Statistiques</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/new/prefix1/55.png</normaloff>:/new/prefix1/55.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <zorder>label_7</zorder>
    <zorder>label_8</zorder>
    <zorder>label</zorder>
    <zorder>pushButton_stat</zorder>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/new/prefix1/48.png</normaloff>:/new/prefix1/48.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Search</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>431</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Titre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_titre"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>431</width>
       <height>381</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Nom Auteur</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_auteur"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_auteur">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Etat</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="checkBox_emprunte">
            <property name="text">
             <string>Emprunté</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_non_emprunte">
            <property name="text">
             <string>Non emprunté</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Nombre de pages sup à</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_minpages"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Nombre de pages inf à</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_maxpages"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QTableView" name="tableView_user">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>30</y>
       <width>441</width>
       <height>431</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_user_status">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>580</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>531</width>
       <height>541</height>
      </rect>
     </property>
     <property name="title">
      <string>Livre Filter</string>
     </property>
     <widget class="QPushButton" name="pushButton_search">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>470</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/new/prefix1/64.png</normaloff>:/new/prefix1/64.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_init">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>470</y>
        <width>60</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/new/prefix1/46.png</normaloff>:/new/prefix1/46.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>10</y>
       <width>541</width>
       <height>541</height>
      </rect>
     </property>
     <property name="title">
      <string>Livre Trouvé</string>
     </property>
     <widget class="QPushButton" name="pushButton_load_user">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>470</y>
        <width>80</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/new/prefix1/42.png</normaloff>:/new/prefix1/42.png</iconset>
      </property>
     </widget>
    </widget>
    <zorder>groupBox_2</zorder>
    <zorder>groupBox</zorder>
    <zorder>layoutWidget_2</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>tableView_user</zorder>
    <zorder>label_user_status</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
